codecov.yaml

```diff
@@ -11,13 +11,25 @@ comment:
   after_n_builds: 1
   show_carryforward_flags: true
 
+github_checks:
+  annotations: true
+
 coverage:
   status:
+    patch:
+      default:
+        target: 60%
+        threshold: 0.1%
+        base: auto
+        only_pulls: true
+        flags:
+          - unit-tests
     project:
       default:
         target: auto
       antrea-unit-tests:
-        target: auto
+        target: 45%
+        threshold: 0.1%
```

Comment:
If set the target to '45%'(or more), the check will always fail as the current test coverage is not as expected.  I think we could just set `target` to 'auto' and set `threshold` to '0%', which makes sure that the unit-test coverage will not decline.

Reply:
Yeah, I think we can set the `target` to `auto`, but we may discuss about `threshold`.

Reply:
The `github_checks annotations` setting seems to work as expected. @antoninbas @tnqn @jianjuns Do you have any comments about `threshold`? Since some code may be hard to cover by adding unit-test cases, the `threshold` allows a drop within a certain range from the previous base commit coverage.

Reply:
there will always be exceptions. We can set it to 0% and when adding unit tests for the new code doesn't make sense, we can just bypass the check. In that case, there will also be a violation for the `patch` target anyway...

Reply:
Good idea, I will update this.